### PR TITLE
Bug fixes

### LINKED ISSUE #49: Check for tables relationships
Currently many tables (esp. in the `staking` module) are related to one another. We should check the relationships between them and ensure they are properly respected using the `REFERENCES` keyword inside column definitions.

### LINKED ISSUE #53: gov/MsgHandler is not parsing any message
## Bug description
in gov/MsgHandler there is no msg passing in and all gov modules message cannot be parsed at the moment.

### PR DESCRIPTION
**Pull request by RiccardoM**
## Description
This PR implements various bug fixes and improvements
- Checked that all table relationships are correct (closes #49) 
- Updated Juno so that all handlers are called even if previous ones fail (closes #53) 

## Checklist
- [ ] Targeted PR against correct branch.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit tests.  
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: schema/08-gov.sql [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 (
-    title             TEXT    NOT NULL,
-    description       TEXT    NOT NULL,
-    proposal_route    TEXT    NOT NULL,
-    proposal_type     TEXT    NOT NULL,
-    proposal_id       DECIMAL NOT NULL PRIMARY KEY,
-    submit_time       TIMESTAMP,
-    deposit_end_time  TIMESTAMP,
-    voting_start_time TIMESTAMP,
-    voting_end_time   TIMESTAMP,
```
Comment (HarleyAppleChoi) [inline schema/08-gov.sql:11]: Sometime `deposit_end_time`,`voting_start_time`,and `voting_end_time` is null when voting is not start yet, isn't it?

**Review by HarleyAppleChoi — APPROVED** [review]
👍👍

File: schema/08-gov.sql (hunk 1 above)
Comment (RiccardoM) [reply schema/08-gov.sql:11]: Ok, didn't know that. Fixed :100:

**Review by HarleyAppleChoi — APPROVED** [review]
👍

**Pull request merged**